screens/AuthScreenController.ts

```diff
@@ -43,12 +44,11 @@ export function useAuthScreen(props: RootRouteProps) {
 
   const { t } = useTranslation('AuthScreen');
 
-  let isBiometricsEnrolled = false;
-  useEffect(() => {
-    async () => {
-      isBiometricsEnrolled = await LocalAuthentication.isEnrolledAsync();
-    };
-  }, []);
+  const fetchIsEnrolled = async () => {
+    const result = await LocalAuthentication.isEnrolledAsync();
+    setIsBiometricsEnrolled(result);
+  };
+  fetchIsEnrolled();
```

Comment:
This is fine for now as a quick band-aid but the biometrics service will need to be refactored into a better state machine later so .